libr/debug/p/native/w32.c

```diff
@@ -272,20 +268,22 @@ static int w32_first_thread(int pid) {
 		eprintf ("w32_thread_list: no thread first\n");
 		return -1;
 	}
-        do {
+        
+	do {
                 /* get all threads of process */
                 if (te32.th32OwnerProcessID == pid) {
 			thid = w32_openthread (THREAD_ALL_ACCESS, 0, te32.th32ThreadID);
-			if (thid == NULL)
+			if (thid == NULL) {
+        			print_lasterr ((char *)__FUNCTION__, "OpenThread");
                                 goto err_load_th;
+			}
 			CloseHandle (th);
 			return te32.th32ThreadID;
 		}
         } while (Thread32Next (th, &te32));
 err_load_th:    
-        if (ret == -1) 
```

Comment:
ret is always -1 (from initialization above). Does not really indicate an error.